packages/mobile/src/invite/saga.ts

```diff
@@ -387,7 +387,8 @@ export function* skipInvite() {
     yield put(setHasSeenVerificationNux(true))
     Logger.debug(TAG + '@skipInvite', 'Done skipping invite')
     ValoraAnalytics.track(OnboardingEvents.invite_redeem_skip_complete)
-    navigateHome()
+    // navigateHome()
+    navigate(Screens.VerificationEducationScreen)
```

Comment:
Should this be gated on the Komenci feature flag? (i.e. if they skip invite and aren't using Komenci, they should be sent to the Home screen?)

Reply:
Yeah this is just a placeholder to allow us to test feeless verification. My subsequent PR for escrow transactions will include the appropriate logic here.